test/v3/controllers/SubmitFinalDeclarationControllerSpec.scala

```diff
@@ -39,9 +42,11 @@ class SubmitFinalDeclarationControllerSpec
     with MockMtdIdLookupService
     with MockSubmitFinalDeclarationService
     with MockSubmitFinalDeclarationParser
+    with MockRetrieveCalculationService
     with MockAuditService
     with MockNrsProxyService
-    with MockIdGenerator {
+    with MockIdGenerator
+    with CalculationFixture {
 
   private val taxYear       = "2020-21"
```

Comment:
Can we do this for all the files for nino, taxYear and calculationId?
```suggestion
  private val taxYear: TaxYear = TaxYear.fromMtd("2019-20")
```